src/dashboard/Dashboard.spec.js

```diff
@@ -1 +1,51 @@
-// Test away
+
+import React from 'react';
+import { render, fireEvent } from 'react-testing-library';
+import renderer from 'react-test-renderer';
+import 'react-testing-library/cleanup-after-each'; 
+
+import Dashboard from './Dashboard';
+
+describe('Dashboard', () => {
+    describe('DEFAULT STATE', () => {
```

Comment:
this is a great way of grouping some tests--instead of checking the default state of each button/display separately, have a describe block just for default state :+1: